.github/workflows/go_release.yml

```diff
@@ -0,0 +1,161 @@
+            target: aarch64-pc-windows-msvc
+            lib: iota_sdk_ffi.dll
+            folder: windows-aarch64
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout iota-rust-sdk
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 #v5.0.0
+
+      - name: Install rust target ${{ matrix.target }}
+        run: rustup target add ${{ matrix.target }}
+
+      - name: Install gcc-aarch64-linux-gnu
+        if: matrix.target == 'aarch64-unknown-linux-gnu'
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+
+      - name: Install gcc-x86-64-linux-gnu
+        if: matrix.target == 'x86_64-unknown-linux-gnu'
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y gcc-x86-64-linux-gnu g++-x86-64-linux-gnu
+
+      - name: Build ${{ matrix.target }} bindings
+        if: matrix.os == 'macos-latest'
+        run: RUSTFLAGS="-C link-args=-Wl,-install_name,@rpath/${{ matrix.lib }}" cargo build -p iota-sdk-ffi --lib --release --target ${{ matrix.target }}
+
+      - name: Build ${{ matrix.target }} bindings
+        if: matrix.os == 'ubuntu-latest'
+        env:
+          CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: /usr/bin/aarch64-linux-gnu-gcc
+          CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER: /usr/bin/x86_64-linux-gnu-gcc
+        run: RUSTFLAGS="-C link-args=-Wl,-rpath,$ORIGIN" cargo build -p iota-sdk-ffi --lib --release --target ${{ matrix.target }}
+
+      - name: Build bindings
+        if: matrix.os == 'windows-latest'
+        run: cargo build -p iota-sdk-ffi --lib --release --target ${{ matrix.target }}
+
+      - name: Prepare upload
+        run: |
+          mkdir -p lib/${{ matrix.folder }}
+          cp target/${{ matrix.target }}/release/${{ matrix.lib }} lib/${{ matrix.folder }}/${{ matrix.lib }}
+
+      - name: Archive ${{ matrix.target }} bindings
+        uses: actions/upload-artifact@330a01c490aca151604b8cf639adc76d48f6c5d4 # v5.0.0
+        with:
+          name: ${{ matrix.target }}
+          path: lib
+
+  release-archives:
+    name: Release archives
+    runs-on: ubuntu-latest
+    needs:
+      - prepare-source-archive
+      - prepare-build-archives
+    steps:
+      - name: Checkout iota-sdk-go
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 #v5.0.0
+        with:
+          repository: iotaledger/iota-sdk-go
+          ssh-key: ${{ secrets.SSH_PRIVATE_KEY_FOR_GO_SDK }}
+          fetch-depth: 0
+
+      - name: Get latest tag from iota-sdk-go
+        id: get-latest-tag
+        run: |
+          LATEST_TAG=$(git tag --sort=-v:refname | head -n 1)
+          if [ -z "$LATEST_TAG" ]; then
+            LATEST_TAG="v0.0.0"
+          fi
+          echo "tag=$LATEST_TAG" >> $GITHUB_OUTPUT
+          echo "Latest tag: $LATEST_TAG"
+
+      - uses: actions-ecosystem/action-bump-semver@34e334551143a5301f38c830e44a22273c6ff5c5 # v1.0.0
```

Comment:
give that step also a name? (also for other steps down below)

Reply:
Done